.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,25 @@
+---
+name: "📖 Documentation"
+about: Add new documentation, or report missing, incorrect or unclear documentation
+title: ''
+labels: "documentation, awaiting triage"
+assignees: ''
+
+---
+
+<!--
+    Please fill in as much of the template below as you’re able to. If you're unsure whether the issue already exists or how to fill in the template, open an issue anyway. Our team will help you to complete the rest.
+
+    Your issue might already exist. If so, add a comment to the existing issue instead of creating a new one. You can find existing issues here: https://github.com/alphagov/govuk-frontend-docs/issues
+
+    We also welcome pull requests from users, if you feel comfortable doing so.
+-->
+
+## Related documentation
```

Comment:
What if there's no related documentation (for example, if a user requests a new page instead of a change to existing content)?

Reply:
Yeah, good point. This template is meant to cover that scenario, but it does lean quite heavily towards assuming the user is suggesting a mistake/gap rather than new documentation. We do suggest that users fill out as much of the template as possible and imply the sections are optional, but I think there's probably more we could do... Welcome to any suggestions!

Reply:
Ah I see - thanks. The template as it is now is probably fine then. (Even if users request a wholly new page, it'll presumably relate to an existing section somehow.) We can always make changes later based on feedback.